src/config/constants/lpAprs.json

```diff
@@ -165,4 +165,4 @@
   "0xb6e34b5c65eda51bb1bd4ea5f79d385fb94b9504": 25.39,
   "0xce383277847f8217392eea98c5a8b4a7d27811b0": 4.36,
   "0xdd5bad8f8b360d76d12fda230f8baf42fe0022cf": 15.06
-}
\ No newline at end of file
+}
```

Comment:
weird. Is git high?

Reply:
I think this means it used to end with `}` before rebase but now it has empty line in the end